packages/cadl-ranch-specs/http/type/primitive/main.tsp

```diff
@@ -0,0 +1,58 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@scenarioService("/type/primitive")
+namespace Type.Primitive;
+
+@route("/string")
+interface String {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
+  @scenario
+  @scenarioDoc("Expect to handle a string value. Mock api will return 'ok'")
+  @get
+  @doc("getStringValue")
+  getStringValue(): string;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
+  @scenario
+  @scenarioDoc("Expect to send a string value. Mock api expect to receive 'test'")
+  @put
+  @doc("putStringValue")
+  putStringValue(@body @doc("_") body: string): void;
+}
+
+@route("/boolean")
+interface Boolean {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
+  @scenario
+  @scenarioDoc("Expect to handle a boolean value. Mock api will return true ")
+  @get
+  @doc("getStringValue")
+  getBooleanValue(): boolean;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
+  @scenario
+  @scenarioDoc("Expect to send a boolean value. Mock api expect to receive 'true'")
+  @put
+  @doc("putBooleanValue")
+  putBooleanValue(@body @doc("_") body: boolean): void;
+}
+
+@route("/unknown")
+interface Unknown {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
+  @scenario
+  @scenarioDoc("Expect to handle a unknown type value. Mock api will return 'test'")
+  @get
+  @doc("getUnknownValue")
+  getUnknownValue(): unknown;
```

Comment:
```suggestion
  get(): unknown;
```

nit all of those shouldn't need to redescribe that, they are already in an interface describing it. Add `@operationGroup` on the interface if you don't want a conflict.

This makes it much worse to look at in teh dashboard otherwise

Reply:
updated.